packages/appeals-service-api/src/services/appeals-case-data.service.js

```diff
@@ -59,6 +59,37 @@ const getAppeals = async (lpaCode) => {
 	return result;
 };
 
+const getAppealByLpaCodeAndCaseRef = async (lpaCode, caseRef) => {
```

Comment:
for clarity, is this the same db query as referred to in aapd-89?

Reply:
Yes